e-commerce/api/server.ts

```diff
@@ -0,0 +1,65 @@
+import { Stripe } from 'stripe'
+import { auth } from '@tensei/auth'
+import { graphql } from '@tensei/graphql'
+import { welcome, tensei, cors, resource, text, textarea, integer, graphQlQuery } from '@tensei/core'
+
+const stripe = new Stripe('sk_test_51JjsxtJnZnb1n2GnbebKQgYe73xTZtT1147z1dLFU9CGmkfbZKvcrW1VX4jA10Wav82slFIGjp9AW6puPURjibMP00z7Fw14Em', {
```

Comment:
Let's use `process.env.STRIPE_SECRET` here. The article should note that this key will be set in environment.

Reply:
Noted